### DISCUSSION TITLE
Getting "AspectJ Internal Error: unable to add stackmap attributes to class +Unsupported class file major version 65" whenever I am adding @Step annotation

### DISCUSSION THREAD

**Discussion by RashmiKalpeshChauhan** [allure-framework/allure2#2507] [Bug Reports] [answered]
Whenever I am adding @Step , it is giving me -AspectJ Internal Error: unable to add stackmap attributes to class 'testcase.*****'. Unsupported class file major version 65

 <maven.compiler.source>21</maven.compiler.source>
 <maven.compiler.target>21</maven.compiler.target>
<allure.version>2.24.0</allure.version>
<aspectj.version>1.9.21.2</aspectj.version>

**Comment by baev** [comment, marked as answer]
The error means that an older AspectJ version is used. The working example: https://github.com/allure-examples/junit5-java-maven

**Reply by RashmiKalpeshChauhan** [reply]
Thanks a lot it is working now
